build.gradle.kts

```diff
@@ -272,6 +272,8 @@ dependencies {
     compileOnly("mineverse.aust1n46:venturechat:2.20.1")
     compileOnly("com.comphenix.protocol:ProtocolLib:4.5.0")
     
+    compileOnly(files("VentureChat-4.0.0.jar"))
```

Comment:
will need to wait for an updated VentureChat build in scarsz' repo

Reply:
Noting that you will want both versions to maintain backwards compatibility.